.gitignore

```diff
@@ -1 +1,2 @@
 smart_proxy_host_reports-*.gem
+/contrib/upload-fixture.conf
```

Comment:
If your script supports config files, why we hide it? I guess it won't harm to have something like `/contrib/upload-fixture.conf.example` file with defaults that can be adjusted and renamed to `/contrib/upload-fixture.conf`.